modules/administration/pages/monitoring.adoc

```diff
@@ -4,6 +4,14 @@
 You can monitor your {productname} environment using Prometheus and Grafana.
 {productname} Server and Proxy are able to provide self-health metrics, or install and manage a limited number of Prometheus exporters on Salt clients.
 
+[NOTE]
+====
+{productname} does not currently support monitoring with Prometheus in private, non-routable networks.
+Prometheus must be able to establish network communications with monitored clients.
```

Comment:
I would write: Since Prometheus fetches metrics based on a 'pull model', it needs to be able to open network connections to the monitored clients.